src/runtime_src/core/tools/xbutil2/CMakeLists.txt

```diff
@@ -87,6 +88,13 @@ if (NOT WIN32)
   target_link_libraries(${XBUTIL2_NAME} PRIVATE pthread uuid dl)
 endif()
 
+# Package xrt sub commands json file for embedded builds
+# This file acts as sample json file and will be removed in future releases
+if (${XRT_NATIVE_BUILD} STREQUAL "no")
+  set(XRT_SUBCOMMANDS_JSON ../common/xrt_subcommands.json)
```

Comment:
Hmm....I though kria team was going to release this file to which we would pick it up.  So I wasn't expecting it to be in the XRT code base.

Reply:
@chvamshi-xilinx talked to kria team and they are asking us to maintain in this release, this file can also act as sample json file.